without the change, mysql-python-connector `8.0.31` is installed in the marvin node
which caused the smoke test failures like below, when the mgmt server is OL10, Debian12 or Suse15

```
Traceback (most recent call last):
  File "/marvin/tests/smoke/test_dynamicroles.py", line 653, in test_role_account_acls_multiple_mgmt_servers
    roleId = self.dbclient.execute("select id from roles where uuid='%s'" % self.role.id)[0][0]
  File "/usr/local/lib/python3.9/site-packages/marvin/dbConnection.py", line 41, in execute
    closing(mysql.connector.connect(host=str(self.host),
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/pooling.py", line 287, in connect
    return MySQLConnection(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/connection.py", line 137, in __init__
    self.connect(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/abstracts.py", line 1108, in connect
    self._open_connection()
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/connection.py", line 544, in _open_connection
    self.set_converter_class(self._converter_class)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/abstracts.py", line 1284, in set_converter_class
    self.converter = convclass(charset_name, self._use_unicode)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/conversion.py", line 136, in __init__
    MySQLConverterBase.__init__(self, charset, use_unicode, str_fallback)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/conversion.py", line 59, in __init__
    self.set_charset(charset)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/conversion.py", line 73, in set_charset
    self.charset_id = CharacterSet.get_charset_info(self.charset)[0]
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/constants.py", line 775, in get_charset_info
    info = cls.get_default_collation(charset)
  File "/usr/local/lib/python3.9/site-packages/mysql/connector/constants.py", line 746, in get_default_collation
    raise ProgrammingError(f"Character set '{charset}' unsupported")
mysql.connector.errors.ProgrammingError: Character set 'utf8' unsupported
```

--- a/tools/marvin/setup.py
+++ b/tools/marvin/setup.py
@@ -46,7 +46,7 @@ setup(name="Marvin",
                 "marvin.sandbox.basic"],
       license="LICENSE.txt",
       install_requires=[
-          "mysql-connector-python <= 8.0.30",
+          "mysql-connector-python >= 8.4.0",
           "requests >= 2.2.1",
           "paramiko >= 1.13.0",
           "nose >= 1.3.3",

